author.rb

```diff
@@ -0,0 +1,16 @@
+class Author
```

Comment:
Good work implementing the add author method, but this is maybe a little confusing. Let me explain how this should work in the application

- When you add a book, the application should ask you for the name of the author
- Once the author is added, you list the books. It should print the name of the books with the name of the author. Something like `The Lord Of The Rings by J.R.R. Tolkien`.